bots/images/scripts/fedora.setup

```diff
@@ -27,6 +27,11 @@ if [ $(uname -m) = x86_64 ]; then
     HAVE_KUBERNETES=1
 fi
 
+HAVE_LIBVIRT_DBUS=
+if [ "$IMAGE" = "fedora-testing" ] || [[ $IMAGE =~ ^fedora\-((2[7-9])|([3-9][0-9]))$ ]]; then
```

Comment:
IIUC, it should filter out just fedora-386 sincefedora-26 is not supported by Cockpit anymore.
What about enabling it on all fedoras except the 386?

Reply:
Yes, basically from the existing images I should exclude  fedora-23-stock, fedora-stock, fedora-atomic and fedora-i386. I change regexp to just exclude these if you think it looks better.

Reply:
Currently fedora.setup is being used for fedora-{27,28,i386,testing}. See `ls -l bots/images/scripts/|grep fedora.setup`.